### ISSUE TITLE
Add APIs for editing SVG values

### ISSUE THREAD

**Issue by bitspittle** [varabyte/kobweb#307]
For some reason, Compose HTML exposes SVG elements (like path, rect, circle) without actually declaring any of their properties.

So I guess we'll have to use extension properties and helper methods instead.

In other words, where now we have to write:

```kotlin
SVG {
    Line(attrs = {
        attr("x1", "3")
        attr("x2", "21")
        attr("y1", "12")
        attr("y2", "12")
    })
}
```
it would be way better to write:

```kotlin
// A
SVG {
   Line {
      x1(3); x2(21)
      y1(12); y2(12)
   }
}
```
or even:
```kotlin
// B
Line(x1 = 3, x2 = 21, y1 = 12, y2 = 12)
```
or even:
```kotlin
// C
Line {
   x(3, 21)
   y(12)
}
```

At a minimum, we should go through the Mozilla docs and add extension method support for all properties within an SVG element scope (case A) above, although adding useful factory methods (case B) would probably be pretty great.

**Comment by binayshaw7777** [comment]
I'm interested in working on this issue.

**Comment by bitspittle** [comment]
This is done and will be released in 0.14.1!
